src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -99,4 +120,20 @@ describe("Dropdown", () => {
       expect(getByRole("combobox")).toHaveAttribute("aria-expanded", "false");
     });
   });
+  it("supports controlled operation", async () => {
+    const user = userEvent.setup();
+    render(<ControlledDropdown />);
+
+    expect(screen.getByRole("option", { name: "Option 1" })).toHaveAttribute(
+      "aria-selected",
+      "true",
+    );
+    await act(() =>
```

Comment:
My understanding of user-event is that it handles all the `act()` stuff internally so it doesn't need to be wrapped in act, instead you just await on user.click() itself, although I'm failing to find any specific docs other than that that's what the examples do: https://testing-library.com/docs/user-event/intro/

Reply:
True :) now seems like a good time to figure out why this test suite spews act warnings without it

Reply:
iirc in EW we had to add `globalThis.IS_REACT_ACT_ENVIRONMENT = true;` to the test setup file

Reply:
I recall in the past this has been caused by the versions of testing-library dependencies being out of sync.